static/js/recent_topics_ui.js

```diff
@@ -303,6 +302,61 @@ function format_topic(topic_data) {
     };
 }
 
+function format_pm(pm_data) {
```

Comment:
Let's extract a function to return the common details between `format_pm` and `format_topic` instead of repeating all the code here.